ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -139,16 +139,12 @@ StackView {
         title: qsTr("Tokens")
 
         buttons: [
-            DisabledTooltipButton {
-                readonly property bool buttonEnabled: root.isPrivilegedTokenOwnerProfile && root.arePrivilegedTokensDeployed
-
-                buttonType: DisabledTooltipButton.Normal
-                aliasedObjectName: "addNewItemButton"
+            StatusButton {
```

Comment:
The conditions here are somehow broken:
- For _Owner_ or _TMaster_ (root.isPrivilegedTokenOwnerProfile), the button should be disabled until `root.arePrivilegedTokensDeployed`, but **no tooltip** shown when disabled.
- For _AdminOnly_  (root.isAdminOnly) it should be always disabled with the tooltip visible

https://github.com/status-im/status-desktop/assets/97019400/67fb8fba-cbfc-4330-97b0-b6fc15317321

Reply:
Nice find! Will fix

Reply:
@noeliaSD fixed! hopefully I got it right this, pls give it a try :) thx

Reply:
Tested again and it works fine!